templates/components/people.html

```diff
@@ -0,0 +1,25 @@
+{% load wagtailimages_tags %}
+
+<div class="people">
+    <h3 class="people__title">{{ title }}</h3>
```

Comment:
```suggestion
    <h3 class="tna-heading tna-heading--s people__title">{{ title }}</h3>
```

Reply:
@ahosgood note that for this and the h2 above, the desktop sizes are different to @nicklee's designs. I can add a custom override in the etna.scss file, or we can leave them different for now and refactor the headings in `tna-frontend` down the line.

Reply:
The font-weights are also different.

Reply:
We _should_ be using the standardised heading sizes that were created for the collection explorer.

The current headings are these: https://nationalarchives.github.io/design-system/styles/typography/#headings

I think we need to get together and align with @nicklee. I'd be keen to avoid having too many heading styles and if we need to update the current ones instead then we can look at doing that at the `tna-frontend` level.